packages/plugins/data-access/coco-data-access/README.md

```diff
@@ -0,0 +1,23 @@
+# @pipcook/plugins-coco-data-access
+
+### Description
+
+This plugin is used to access coco data. This means that whatever data provided by data-collect (no matter whether you chooose pascal voc, coco or other formats), the data will be transformed to coco dataset and flow into model-related plugins.
```

Comment:
We need to have a link to the coco dataset.

Reply:
done